package.json

```diff
@@ -188,20 +188,28 @@
             }
           }
         },
-        "initialConfigurations": [
```

Comment:
Why did you remove `initialConfigurations`?

Reply:
we can reserve `initialConfigurations` and add `configurationSnippets` like the third suggestion in https://github.com/felixfbecker/vscode-php-debug/issues/318.

Reply:
fwiw
```
        "configurationSnippets": [
          {
            "label": "PHP: Listen for XDebug",
            "description": "Listening for XDebug",
            "body": {
              "name": "Listen for XDebug",
              "type": "php",
              "request": "launch",
              "port": 9000
            }
          },
          {
            "label": "PHP: Launch currently open script",
            "description": "Launch currently open script",
            "body": {
              "name": "Launch currently open script",
              "type": "php",
              "request": "launch",
              "program": "^\"${1:\\${file\\}}\"",
              "cwd": "^\"${2:\\${fileDirname\\}}\"",
              "port": 9000
            }
          }
        ]
```